samples/bidirectional-chat/csharp/Function.cs

```diff
@@ -0,0 +1,112 @@
+        {
+            return connectionInfo;
+        }
+
+        [FunctionName(nameof(Connected))]
+        public async Task Connected([SignalRTrigger]InvocationContext invocationContext, ILogger logger)
+        {
+            await Clients.All.SendAsync(NewConnectionTarget, new NewConnection(invocationContext.ConnectionId));
+            logger.LogInformation($"{invocationContext.ConnectionId} has connected");
+        }
+
+        [FunctionName(nameof(Broadcast))]
+        public async Task Broadcast([SignalRTrigger]InvocationContext invocationContext, string message, ILogger logger)
+        {
+            await Clients.All.SendAsync(NewMessageTarget, new NewMessage(invocationContext, message));
+            logger.LogInformation($"{invocationContext.ConnectionId} broadcast {message}");
+        }
+
+        [FunctionName(nameof(SendToGroup))]
+        public async Task SendToGroup([SignalRTrigger]InvocationContext invocationContext, string groupName, string message)
+        {
+            await Clients.Group(groupName).SendAsync(NewMessageTarget, new NewMessage(invocationContext, message));
+        }
+
+        [FunctionName(nameof(SendToUser))]
+        public async Task SendToUser([SignalRTrigger]InvocationContext invocationContext, string userName, string message)
+        {
+            await Clients.User(userName).SendAsync(NewMessageTarget, new NewMessage(invocationContext, message));
+        }
+
+        [FunctionName(nameof(SendToConnection))]
+        public async Task SendToConnection([SignalRTrigger]InvocationContext invocationContext, string connectionId, string message)
+        {
+            await Clients.Client(connectionId).SendAsync(NewMessageTarget, new NewMessage(invocationContext, message));
+        }
+
+        [FunctionName(nameof(JoinGroup))]
+        public async Task JoinGroup([SignalRTrigger]InvocationContext invocationContext, string connectionId, string groupName)
+        {
+            await Groups.AddToGroupAsync(connectionId, groupName);
+        }
+
+        [FunctionName(nameof(LeaveGroup))]
+        public async Task LeaveGroup([SignalRTrigger]InvocationContext invocationContext, string connectionId, string groupName)
+        {
+            await Groups.RemoveFromGroupAsync(connectionId, groupName);
+        }
+
+        [FunctionName(nameof(JoinUserToGroup))]
+        public async Task JoinUserToGroup([SignalRTrigger]InvocationContext invocationContext, string userName, string groupName)
+        {
+            await UserGroups.AddToGroupAsync(userName, groupName);
+        }
+
+        [FunctionName(nameof(LeaveUserFromGroup))]
+        public async Task LeaveUserFromGroup([SignalRTrigger]InvocationContext invocationContext, string userName, string groupName)
+        {
+            await UserGroups.RemoveFromGroupAsync(userName, groupName);
+        }
+
+        [FunctionName(nameof(Disconnect))]
+        public void Disconnect([SignalRTrigger]InvocationContext invocationContext)
```

Comment:
Should be `disconnected`?

Reply:
Use another PR to do that change including `onconnected`